src/fields/CheckBoxGroup.jsx

```diff
@@ -1,7 +1,11 @@
-var React = require("react");
-var cleanProps = require("./clean-props");
+import React from 'react';
```

Comment:
if we only need `React` for `Component` we could `import { Component } from 'react'` and then do `export default class CheckBoxGroup extends Component`